### PR TITLE
chore(liveness): Update docs for nolights feature

### PR DESCRIPTION
**Pull request by esauerbo**
#### Description of changes
- Wherever photosensitivity warning is mentioned, mention it only works with movement + light challenge 
- Document new flow

New docs should have the same links for setting up backend so no changes there

Screenshot 2024-06-06 at 12 51 42 PM
Screenshot 2024-06-06 at 12 53 02 PM

#### Checklist

- [ ] Have read the [Pull Request Guidelines](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md)
- [ ] PR description included
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] `yarn test` passes and tests are updated/added
- [ ] No side effects or [`sideEffects`](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CONTRIBUTING.md#code-standards) field updated

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/src/pages/[platform]/connected-components/liveness/index.page.mdx [hunk 1]
```diff
@@ -29,7 +29,12 @@ export async function getStaticProps() {
   return { props: {} }
 }
 
-Amplify UI FaceLivenessDetector is powered by [Amazon Rekognition Face Liveness](https://aws.amazon.com/rekognition/). The following screenshots shows an example of FaceLivenessDetector in action.
+The Amplify UI FaceLivenessDetector is powered by [Amazon Rekognition Face Liveness](https://aws.amazon.com/rekognition/face-liveness/). The FaceLivenessDetector supports two types of verification challenges:
+
+ - The **Face Movement and Light Challenge** consists of a series of head movements and colored light tests.
+ - The **Face Movement Challenge** consists only of a series of head movements.
```
Comment (esauerbo) [inline docs/src/pages/[platform]/connected-components/liveness/index.page.mdx:35]: Added a followup to link to the amplify docs here bc the documentation for the new challenge hasn't been released yet

File: docs/src/pages/[platform]/connected-components/liveness/customization/Customizationi18n.tsx [hunk 2]
```diff
@@ -6,8 +6,8 @@ const dictionary = {
   // use default strings for english
   en: null,
   es: {
-    photosensitivyWarningHeadingText: 'Advertencia de fotosensibilidad',
```
Comment (reesscot) [inline docs/src/pages/[platform]/connected-components/liveness/customization/Customizationi18n.tsx:9]: Is this a breaking change? Can you confirm it's not written this way in the codebase or that we have a fallback to use the correct spelling version?

Comment (reesscot) [reply docs/src/pages/[platform]/connected-components/liveness/customization/Customizationi18n.tsx:9]: This is fixed now, right?

Comment (esauerbo) [reply docs/src/pages/[platform]/connected-components/liveness/customization/Customizationi18n.tsx:9]: The fix is merged but not released. This usage is what currently works on latest, and the fix is to make incorrect and correct spellings both work

File: docs/src/pages/[platform]/connected-components/liveness/react-props.ts [hunk 3]
```diff
@@ -133,4 +133,10 @@ export const FACE_LIVENESS_DETECTOR_ERROR_STATES = [
       'Indicates that the user attempted to switch to using landscape mode which is not supported.',
     type: `Error`,
   },
+  {
+    name: `MULTIPLE_FACES_ERROR`,
+    description:
+      'Indicates that the multiple faces have been detected within the frame.',
```
Comment (reesscot) [inline docs/src/pages/[platform]/connected-components/liveness/react-props.ts:139]:
```suggestion
      'Indicates that multiple faces have been detected within the frame.',
```

**Review by reesscot — APPROVED** [review]

**Review by thaddmt — APPROVED** [review]

**Pull request merged**